src/main/java/org/sciborgs1155/robot/Robot.java

```diff
@@ -263,6 +269,8 @@ private void configureBindings() {
                 .deadlineWith(Commands.idle(shooter)))
         .toggleOnTrue(led.raindrop());
 
+    operator.y().onTrue(shooting.feedToAmp(x, y));
```

Comment:
Maybe this should be `.whileTrue`? In simulation, the robot continues pointing at the feeding target area, even after the note is shot.

Reply:
It also seems to sometimes make it only shoot once in simulation and not again?

Reply:
Shooting only once is cuz of feeder code